cpp/include/cudf_test/iterator_utilities.hpp

```diff
@@ -77,7 +77,7 @@ static auto iterator_with_null_at(Iter index_start, Iter index_end)
  * @param indices The indices for which the validity iterator must return `false` (i.e. null)
  * @return auto Validity iterator
  */
-static auto iterator_with_null_at(cudf::host_span<cudf::size_type const> const& indices)
+[[maybe_unused]] auto iterator_with_null_at(cudf::host_span<cudf::size_type const> const& indices)
```

Comment:
Not related to the change here, feel free to ignore.
```suggestion
[[maybe_unused]] auto iterator_with_null_at(cudf::host_span<cudf::size_type const> indices)
```

Reply:
I like that. Thanks.